src/clj/rems/handler.clj

```diff
@@ -61,7 +60,10 @@
     (redirect "/redirect"))
 
   (GET "/favicon.ico" []
-    (redirect "/img/favicon.ico")))
+    (redirect "/img/favicon.ico"))
+
+  (GET "/entitlements.csv" [] ; DEPRECATED: legacy url redirect 
+    (redirect "/api/entitlements/csv")))
```

Comment:
i'll add reminder ticket about deprecation if this is a viable solution. redirect is quite dependent on clients following the location response header.